### PR TITLE
fix(link account): add cancel button to link account flow

### PR DESCRIPTION
**Pull request by stephenliang**
Currently, users who enter the link account flow gets "locked" into that state. If a user wishes to navigate out of the link account flow or cancel the flow (because they realize this is the wrong account), the user is unable to with no options to log out or cancel. This change adds a cancel button to allow the user to escape from the flow.

## Links

- [jira ticket](https://codedotorg.atlassian.net/browse/P20-998)
- [jira ticket](https://codedotorg.atlassian.net/browse/P20-997)

## Testing story

Screenshot from 2024-06-18 10-49-54
Screenshot from 2024-06-18 10-49-48

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/ui/lti/link/LtiLinkAccountPage/index.tsx [hunk 1]
```diff
@@ -4,10 +4,26 @@ import React from 'react';
 import NewAccountCard from './cards/NewAccountCard';
 import ExistingAccountCard from './cards/ExistingAccountCard';
 import WelcomeBanner from './WelcomeBanner';
+import {buttonColors, Button} from '@cdo/apps/componentLibrary/button';
+import i18n from '@cdo/locale';
+import {navigateToHref} from '@cdo/apps/utils';
 
 const LtiLinkAccountPage = () => {
+  const handleCancel = () => {
+    navigateToHref(`/users/cancel`);
```
Comment (carl-codeorg) [inline apps/src/lib/ui/lti/link/LtiLinkAccountPage/index.tsx:13]: Nice job tracking down the route for this!

**Review by carl-codeorg — COMMENTED** [review]
This LGTM but I think we should rope in @alivira to confirm the styling and placement of the buttons. UX has specific colors and stuff we may need to use.

**Review by carl-codeorg — APPROVED** [review]
:shipit:

**Comment by alivira** [comment]
updated UI lgtm!

**Pull request merged**